src/bid/encoding.rs

```diff
@@ -125,13 +133,27 @@ mod tests {
         let value: u64 = (&mut rand::thread_rng())
             .gen_range(crate::V_RAW_MIN, crate::V_RAW_MAX);
         let value = JubJubScalar::from(value);
+        let pk_r = AffinePoint::from(
```

Comment:
Should generate a random `dusk_pki::PublicSpendKey` and then generate PKr with `PublicSpendKey::gen_stealth_address`

Reply:
is it really important if it's for testing?

Reply:
@vlopes11 ping

Reply:
Hmm dunno, but its more consistent with the general idea.